pkg/compilers/names.go

```diff
@@ -45,6 +45,7 @@ var (
 	RUMP_JAVA_QEMU       = compilerName("rump", "java", "qemu")
 
 	OSV_JAVA_XEN        = compilerName("osv", "java", "xen")
+	OSV_JAVA_OPENSTACK  = compilerName("osv", "java", "openstack")
```

Comment:
My thinking was to group cloud providers, but I see your point as well; moving this down.